test/test.c

```diff
@@ -5466,6 +5466,31 @@ test_natsSrvVersionAtLeast(void)
     natsConnection_Destroy(nc);
 }
 
+static void
+test_natsFormatStringArray(void)
+{
+    natsStatus s;
+    char *str = NULL;
+
+    test("Check empty: ");
+    s = nats_formatStringArray(&str, NULL, 0);
+    testCond((s == NATS_OK) && (str != NULL) && (strcmp(str, "[]") == 0));
+    NATS_FREE(str);
+    str = NULL;
+
```

Comment:
Since you account for `NULL` string in the array (and format as `(null)`), should you add a test for that?

Reply:
just did